src/SDL_SetError.3

```diff
@@ -49,18 +68,15 @@ a.
 T{
 .nf
 int errorCode = 0;
-...
+ ...
```

Comment:
Hey! Just tiny nit here, would you remove leading space here. This line and below.

Reply:
Oh yeah I've got an issue with that. Without a whitespace the dots would not show in my manual. I assume this is because dots have a special meaning in roff. I tried several things to escape the leading dot but none worked.
Do you have any idea about that?

Reply:
I re-check again. You're right. On my end now, it won't render properly and also shout out error as follows before your changes

```
$ man --warning /path/SDL_SetError.3 >> /dev/null
troff: <standard input>:71: warning: macro '..' not defined
```

Not sure why that time I saw it properly rendered.
Anyway, I'll fix this issue separately. Thanks!